open-science-projects.md

```diff
@@ -224,7 +224,9 @@ Students also learn about the process of writing a collaborative paper by attend
 ## Mistakes and benefits
 
 We have made plenty of mistakes.
-Students have spent weeks stuck on problems that other team members know how to solve, have written large chunks of code that aren't possible to incorporate into the larger project, and have cobbled together their own (undocumented) data structures.
+Students have spent weeks stuck on problems that other team members know how to solve, have written large chunks of code that aren't possible to incorporate into the larger project, and have cobbled together their own (undocumented and barely usable) data structures.
```

Comment:
Perhaps "undocumented and difficult to use"?

Reply:
Yes, or we could go back to just “undocumented”